packages/trpc/server/routers/viewer/organizations/createTeams.handler.ts

```diff
@@ -18,10 +18,14 @@ type CreateTeamsOptions = {
 export const createTeamsHandler = async ({ ctx, input }: CreateTeamsOptions) => {
   const { teamNames, orgId } = input;
 
-  const organization = await prisma.team.findFirst({ where: { id: orgId }, select: { metadata: true } });
+  const organization = await prisma.team.findFirst({
+    where: { id: orgId },
+    select: { slug: true, metadata: true },
+  });
   const metadata = teamMetadataSchema.parse(organization?.metadata);
 
-  if (!metadata?.requestedSlug) throw new TRPCError({ code: "BAD_REQUEST", message: "no_organization" });
+  if (!metadata?.requestedSlug || !organization?.slug)
```

Comment:
Not only rely on `requestedSlug` to show an error, when no team-billing is enabled, org `slug` is the one defined.

Reply:
I am still getting the error, should that be && maybe?

Reply:
Sharp eyes :)